### PR TITLE
#1326 - Removed duplicate literals warning.

### PR DESCRIPTION
**Pull request by vzurauskas**
PR for #1326 

Removed duplicate String literals.

### CODE REVIEW

**Comment by 0crat** [comment]
This pull request [#1330](https://github.com/yegor256/cactoos/issues/1330) is assigned to @victornoel[/z](https://www.0crat.com/u/victornoel), here is [why](https://www.0crat.com/footprint/C63314D6Z/0ff56da9-35b2-4b39-a6b0-3a749b73ac1c); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @paulodamaso[/z](https://www.0crat.com/u/paulodamaso) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be no monetary reward for this job

**Comment by victornoel** [comment]
@0crat refuse

**Comment by victornoel** [comment]
@vzurauskas @paulodamaso ah sorry, I made a mistake, I didn't want to refuse ^^ if needed you can re-assign me if there is nobody else

**Comment by 0crat** [comment]
> @0crat refuse ([here](https://github.com/yegor256/cactoos/issues/1330#issuecomment-601858504))

@victornoel The user @victornoel[/z](https://www.0crat.com/u/victornoel) resigned from [#1330](https://github.com/yegor256/cactoos/issues/1330), please stop working. Reason for job resignation: Order was cancelled

**Comment by 0crat** [comment]
> @0crat refuse ([here](https://github.com/yegor256/cactoos/issues/1330#issuecomment-601858504))

@victornoel Job refused in 2 hours - no penalty, see [§6](http://www.zerocracy.com/policy.html#6)

**Comment by 0crat** [comment]
This pull request [#1330](https://github.com/yegor256/cactoos/issues/1330) is assigned to @victornoel[/z](https://www.0crat.com/u/victornoel), here is [why](https://www.0crat.com/footprint/C63314D6Z/28931e2b-7e76-4ada-bc5b-321d0d4a59f8); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @paulodamaso[/z](https://www.0crat.com/u/paulodamaso) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be no monetary reward for this job

File: src/test/java/org/cactoos/iterable/SkippedTest.java [hunk 1]
```diff
@@ -120,19 +124,18 @@ public void skippedMoreThanExists() {
     @Test
     @SuppressWarnings("PMD.AvoidDuplicateLiterals")
```
Comment (victornoel) [inline src/test/java/org/cactoos/iterable/SkippedTest.java]: @vzurauskas you can remove this now I suppose

Comment (vzurauskas) [reply src/test/java/org/cactoos/iterable/SkippedTest.java]: @victornoel Removed.

**Comment by victornoel** [comment]
@paulodamaso seems good to merge

**Comment by paulodamaso** [comment]
@victornoel @vzurauskas Thanks!

**Comment by paulodamaso** [comment]
@rultor merge

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 11@ef7cb9a334af, thread: PQ-C63314D6Z
com.jcabi.xml.StrictXML[124] java.lang.IllegalArgumentException: 1 error(s) in XML document: -1:-1: Duplicate unique value [gh:yegor256/cactoos#1330] declared for identity constraint of element "speed".
```

[0.52.4](https://github.com/zerocracy/farm): CID: [99dabd6a-4070-4937-ae04-eafee0a50295](https://www.0crat.com/footprint/C63314D6Z/99dabd6a-4070-4937-ae04-eafee0a50295), Type: "Order was finished"

**Comment by 0crat** [comment]
Code review was too long (8 days), architects (@paulodamaso) were penalized, see [§55](http://www.zerocracy.com/policy.html#55)

**Comment by 0crat** [comment]
@sereshqua[/z](https://www.0crat.com/u/sereshqua) please review this job completed by @victornoel[/z](https://www.0crat.com/u/victornoel), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by sereshqua** [comment]
@victornoel please make sure you will at least 3 issues during next CR, thx :)

**Comment by victornoel** [comment]
@sereshqua noted :)

**Comment by sereshqua** [comment]
@0crat quality acceptable

**Pull request merged**